calendar/utils/kvstore/plugin_store_test.go

```diff
@@ -0,0 +1,323 @@
+package kvstore
+
+import (
+	"testing"
+	"time"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/plugin"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+type MockPluginAPI struct {
+	plugin.API
+	mock.Mock
+}
+
+func (m *MockPluginAPI) KVGet(key string) ([]byte, *model.AppError) {
```

Comment:
What was the need for mocking these?

Reply:
To implement the PluginAPI. Let me know if we have any better option for that

Reply:
But we already have a method for that by MM right? i.e. plugintest

Reply:
pluginTest seems to work for fine type `pluginAPI`, but the API in this file is
`type pluginStore struct {
	api        plugin.API
	ttlSeconds int64
}`